deploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+# Free Travis version does not support SSH key pairs, thus using encrypted files for SSH deploy key
+# https://docs.travis-ci.com/user/encrypting-files/
+openssl aes-256-cbc -K $encrypted_989f4ea822a6_key -iv $encrypted_989f4ea822a6_iv -in ansible/files/travis/master_key.enc -out master_key -d
```

Comment:
This line says `-out master_key`. Lines after say `/tmp/deploy_rsa` (see lines 8 and 9).
It looks like names shall be the same? (It looks copy-paste from https://github.com/aeternity/epoch/pull/362/files?w=1#diff-35ea51baf1fe7f0142ad5f950855dde0R83)

And also, how does this related to `ansible/files/travis/master_key.pub`?

Reply:
fixed in 0f71228817b545dd527531951b40a74d1373dd06

I'm tracking master_key.pub just for the sake of completeness. It's not used yet in the automation.